addons/io_scene_gltf2/blender/exp/gltf2_blender_generate_extras.py

```diff
@@ -16,6 +16,37 @@
 import bpy
 from io_scene_gltf2.blender.com import gltf2_blender_json
 
```

Comment:
for [PEP8](https://www.python.org/dev/peps/pep-0008/) conformity, there have to be 2 empty lines between imports and the first `def`